core/src/main/java/io/grpc/Status.java

```diff
@@ -44,7 +44,7 @@
 
 /**
  * Defines the status of an operation by providing a standard {@link Code} in conjunction with an
- * optional descriptive message. Instances of {@code Status} are created by starting with the
+ * optional descriptive message. Instances of {@link Status} are created by starting with the
```

Comment:
I do think this one was on purpose, because it being a @ link adds no value and in a way is just blue-link noise.